### DISCUSSION TITLE
Thanks so much! But can I ask for column sorting too?

### DISCUSSION THREAD

**Discussion by AndrewMac** [craftcms/console#70] [Craft Console]
First off, I just want to say thanks so much for the new Organizations feature. It's been on my wishlist ever since I first started using id.craftcms.com. Before Organizations I had to fake it all with notes and filters and stuff …

But can I just tell you the other big feature that's always been on my wishlist? The ability to sort the cms/plugin tables by column. I'm sure you know what I mean, but just in case: it would be great to be able to click on one of the column headers ("Expires on", for example) to sort the entries by that column. Clicking again would reverse the sort order, etc.

The primary use case for me is that I could then quickly sort an organization's plugins by date and then advise them which renewals will be coming up in the coming month(s).

Thanks for listening … but most of all, thanks once again for Organizations!

Andrew Macpherson

**Comment by brandonkelly** [comment]
Agree we should add this!

**Reply by AndrewMac** [reply]
Thanks so much, so glad to hear you're interested in adding this!

**Reply by HannahDeWachter** [reply]
@brandonkelly Is there any news on the column sorting? 😄
